app/Http/Controllers/VotesController.php

```diff
@@ -9,7 +9,7 @@
 use VotingApp\Http\Requests\VoteRequest;
 use VotingApp\Models\Candidate;
 use VotingApp\Models\Vote;
-use Northstar;
+use DoSomething\Gateway\Northstar;
```

Comment:
You "use" a namespace to allow it to be accessed later without the fully qualified path. Since you never use a `Northstar` class below, there's no need to import it.